tracer/tracer.go

```diff
@@ -64,6 +65,12 @@ const (
 	probProfilingDisable = -1
 )
 
+const (
+	// OffCpuThresholdMax defines the upper bound for the off-cpu profiling
+	// threshold.
+	OffCPUThresholdMax = support.OffCPUThresholdMax
+)
+
```

Comment:
No need to introduce additional level of indirection here

```suggestion
const (
	// OffCpuThresholdMax defines the upper bound for the off-cpu profiling
	// threshold.
	OffCPUThresholdMax = support.OffCPUThresholdMax
)
```